examples/advanced/webpack.config.js

```diff
@@ -1,6 +1,12 @@
 // NOTE: To use this example standalone (e.g. outside of deck.gl repo)
 // delete the local development overrides at the bottom of this file
 
+// hack for node 17+
```

Comment:
Add a description why this is needed

Reply:
webpack uses old MD4 hashing which is not available in node 17+ anymore (removed from openssl lib)